### DISCUSSION TITLE
usoc进度回退问题

### DISCUSSION THREAD

**Discussion by Aurora-Simba** [hooke007/mpv_PlayKit#349] [Q&A 问答] [answered]
点按进度条向右可快进，向左点一下无效，并会卡住，点两下或拖动进度条可回退
已排查为硬解造成，软解无问题
配置如下

 vo = gpu-next
 gpu-context = d3d11
 d3d11-exclusive-fs = no
 d3d11-flip = no

 hwdec = qsv-copy,auto
 hwdec-codecs = all

 vd-lavc-dr = auto

**Comment by hooke007** [comment, marked as answer]
qsv不在推荐列表上，你用它的理由是什么呢。而且这和ui有什么关系。。。
如果这能在no-config的条件下复现，应该向mpv报告
